db/migrate/20230520180121_create_orders.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOrders < ActiveRecord::Migration[7.0]
+  def change
+    create_table :orders do |t|
+      t.string :product_name
+      t.integer :product_count
+      t.references :customer_id, null: false, foreign_key: true, type: integer
```

Comment:
I'm also surprised that `customer_id` works. Normally they're done this way:

```suggestion
      t.references :customer, null: false, foreign_key: true
```